docs/docs/advanced/setNativeProps.mdx

```diff
@@ -74,6 +73,8 @@ import SetNativePropsSrc from '!!raw-loader!@site/src/examples/SetNativeProps';
 
 - You should always reach for [`useAnimatedStyle`](docs/core/useAnimatedStyle) and [`useAnimatedProps`](docs/core/useAnimatedProps) first when animating styles or properties.
 
+- `setNativeProps` is implemented only on the [UI thread](/docs/fundamentals/glossary#ui-thread). When using `setNativeProps` inside [event handlers](https://react.dev/learn/responding-to-events#adding-event-handlers), it has to be wrapped with the [`runOnUI`](/docs/threading/runOnUI) function.
```

Comment:
There's no point using Reanimated's `setNativeProps` via `runOnUI` from the RN runtime since you can use original RN's `setNativeProps` from here.